VideoLocker/res/layout/player_controller.xml

```diff
@@ -1,224 +1,224 @@
             android:contentDescription="@string/app_name"
-            tools:targetApi="17"
-            />
-                
-    </RelativeLayout>
+            app:iconColor="@color/edx_grayscale_neutral_white_t"
+            app:iconName="fa_facebook_square"
+            tools:targetApi="17" />
+
+    </LinearLayout>
 
     <LinearLayout
         android:id="@+id/panel_bottom"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_alignParentBottom="true"
-        android:layoutDirection="ltr"
+        android:layout_gravity="bottom"
         android:background="#CC000000"
-        android:orientation="vertical"
-        android:padding="7dp" >
+        android:gravity="center_vertical"
+        android:orientation="horizontal"
+        android:padding="7dp">
+
+        <org.edx.mobile.third_party.iconify.IconButton
+            android:id="@+id/rew"
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_marginEnd="8dp"
+            android:layout_marginLeft="4dp"
+            android:layout_marginRight="8dp"
+            android:layout_marginStart="4dp"
+            android:background="@null"
+            android:contentDescription="@string/app_name"
+            app:iconColor="@color/edx_grayscale_neutral_white_t"
+            app:iconName="fa_history"
+            android:scaleType="fitXY"
+            tools:targetApi="17" />
+
+        <SeekBar
+            android:id="@+id/mediacontroller_progress"
+            android:layout_width="0dp"
+            android:layout_height="24dp"
+            android:layout_marginEnd="6dp"
+            android:layout_marginRight="6dp"
+            android:layout_weight="1"
+            android:background="@drawable/ic_seek_background"
+            android:max="100"
+            android:maxHeight="24dp"
+            android:minHeight="24dp"
+            android:paddingEnd="0dp"
+            android:paddingLeft="0dp"
+            android:paddingRight="0dp"
+            android:paddingStart="0dp"
+            android:progress="30"
+            android:progressDrawable="@drawable/player_seekbar_progress"
+            android:secondaryProgress="70"
+            android:thumb="@drawable/player_seekbar_thumb"
+            android:thumbOffset="0dp"
+            tools:targetApi="17" />
 
-        <LinearLayout
-            android:layout_width="match_parent"
+        <org.edx.mobile.view.custom.ETextView
+            android:id="@+id/time_current"
+            style="@style/semibold_white_text"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:gravity="center"
-            android:orientation="horizontal"
+            android:layout_gravity="center"
+            android:paddingEnd="0dp"
+            android:paddingLeft="4dip"
+            android:paddingRight="0dp"
+            android:paddingStart="4dip"

```

Comment:
Thats what Ctrl+Alt+L does. Nevertheless, I've moved paddings and margins to styles and there they are properly ordered.